packages/trip-details/src/index.js

```diff
@@ -182,3 +184,9 @@ TripDetails.defaultProps = {
   routingType: "ITINERARY",
   timeOptions: null
 };
+
+export default TripDetails;
+
+// Rename styled components for export
+const Styled = S;
```

Comment:
I would `import * as Styled` and remove this statement and comment above.

Reply:
See previous comments, have updated the export style